style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Inter", sans-serif;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 15px;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+header h2 {
+  font-weight: 600;
+}
+
+header .fa-bars {
+  font-weight: 500;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.home-page {
```

Comment:
- [ ] The previous reviewer mentioned that you need to use `flexbox` to align this `headline section`. The CSS class you applied to the section is `home-page`, therefore you are to apply the flexbox to this class. Kindly apply display flex to this CSS class.